### ISSUE TITLE
broken compilation in node cljs

### ISSUE THREAD

**Issue by selfsame** [selfsame/pdf#22]
```clj
(defpdfn huh)

(pdfn huh [^even? a ^odd? b])
;WARNING: Can't take value of macro cljs.core/and at line 1 

(inspect huh)
;(set! huh (fn ([_260 _261] (if and (huh2_1323207358 _260 _261)))))
```

**Comment by selfsame** [comment]
```clj
(inspect huh :ast)
([if
  (and (even? _260) (odd? _261))
  [huh2_1323207358 _260 _261] 
  :pdfn.core/nf])
```

**Comment by selfsame** [comment]
ast is the same

**Comment by selfsame** [comment]
problem is in `ast->code` where `(list? (cons '+ (list 1 2)))` is false in `:clj`, and node cljs isn't using `:clj` to compile

**Comment by selfsame** [comment]
fixed with 396a2c51900bdcd6d4be91e482afdb7808088cd7
